### PR TITLE
chore: check format of 'BREAKING CHANGE:' notice

### PR DESCRIPTION
**Pull request by rix0rrr**
I myself keep on forgetting whether it's spelled 'BREAKING CHANGE:' or
'BREAKING CHANGES:', and maybe whether or not to put a `(topic)` there,
that I prefer a machine checks this for me.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by iliapolo — COMMENTED** [review]
Maybe put this in a different check? i.e *Body Format* or something?

https://github.com/aws/aws-cdk/blob/master/.github/workflows/pr-linter.yml

**Comment by iliapolo** [comment]
This is great BTW :)

File: tools/prlint/index.js [hunk 1]
```diff
@@ -76,6 +76,23 @@ function hasLabel(issue, labelName) {
...
+ * Check this by looking for something that most likely was intended
+ * to be said note, but got misspelled as "BREAKING CHANGES:" or
+ * "BREAKING CHANGES(module):"
+ */
+function validateBreakingChangeFormat(body) {
+    const re = /^BREAKING.*$/m;
+    const m = re.exec(body);
+    if (m) {
+        if (!m[0].startsWith('BREAKING CHANGE: ')) {
+            throw new LinterError(`Breaking changes should be indicated by starting a line with \'BREAKING CHANGE: \', variations are not allowed. (found: '${m[0]}')`);
```
Comment (njlynch) [inline tools/prlint/index.js:91]:
```suggestion
            throw new LinterError(`Breaking changes should be indicated by starting a line with 'BREAKING CHANGE: ', variations are not allowed. (found: '${m[0]}')`);
```

**Review by njlynch — APPROVED** [review]

**Review by iliapolo — CHANGES_REQUESTED** [review]
If this check stays here, then we should rename the job and and check to a generic `validate` method, since it no longer refers just to mandatory changes.

Another option is to put this in another job, but I can see why this has little value and requires more work.

**Review by iliapolo — APPROVED** [review]

**Pull request merged**